### PR TITLE
MAUI-1858-UG Content for CartesianChart SelectionBehavior 

### PR DESCRIPTION
**Pull request by MoneeshramDhanabal**
### Feature description ###
Preparing the UG document for the new updated Selection Behavior in the SfCartesianChart.

### Solution description ###

UG document guides the user to understand the types of Selection that can be achieved using Selection behavior.

### Output screenshots ###

UG Demo image for DataPointSelection :
maui_chart_segment_selection

UG Demo image for SeriesSelection :
maui_chart_seriesSelection

UG Demo image for MultiSelectionSelection :
maui_multi_dataPoint_selectionmaui_multi_seriesselection

### CODE REVIEW

File: MAUI/Cartesian-Charts/Selection.md [hunk 1]
```diff
@@ -84,44 +88,125 @@ Cartesian chart provides support to select a point programmatically on a chart u
...
 
 {% endtabs %}
 
-![Initial selection rendering support in MAUI Chart](Selection_images/maui_chart_selection_with_index.png)
+N> The [ClearSelection ()]() method can be performed in [SeriesSelectionBehavior]() by using the SfCartesianChart SelectionBehavior property. 
 
 ## Events
 
-The following selection events are available in the [SfCartesianChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1).
+The following public Chart Selection Events are available in [ChartSelectionBehavior]() and are common for both [DataPointSelectionBehavior]() and [SeriesSelectionBehavior]().
```
Comment (OnyimbiNelson) [inline MAUI/Cartesian-Charts/Selection.md:198]: The following public Chart Selection Events are available in [ChartSelectionBehavior]() and are common for both [DataPointSelectionBehavior]() and [SeriesSelectionBehavior]():

Comment (MoneeshramDhanabal) [reply MAUI/Cartesian-Charts/Selection.md:200]: I have changed the punctuations as mentioned

Comment (MoneeshramDhanabal) [reply MAUI/Cartesian-Charts/Selection.md:200]: I have also resolved this punctuation change.

File: MAUI/Cartesian-Charts/Selection.md [hunk 2]
```diff
@@ -84,44 +88,125 @@ Cartesian chart provides support to select a point programmatically on a chart u
...
 
 ### SelectionChanging
 
-The [SelectionChanging](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartBase.html#Syncfusion_Maui_Charts_ChartBase_SelectionChanging) event occurs before the data point is being selected. This is a cancelable event. This argument contains the following information.
+The [SelectionChanging]() event is triggered before any data point has been selected. This event is cancelable because it inherits CancelEventArgs, which has a public property [Cancel]() that holds a Boolean value indicating whether to continue the selection or not, and the properties below are contained in the following event arguments.
```
[the remark this answers is not in the document]
Comment (MoneeshramDhanabal) [reply MAUI/Cartesian-Charts/Selection.md]: I have also resolved this punctuation change.

File: MAUI/Cartesian-Charts/Selection.md [hunk 3]
```diff
@@ -84,44 +87,126 @@ Cartesian chart provides support to select a point programmatically on a chart u
...
+{% endtabs %}
+
+![Multiple SeriesSelection support in MAUI SfCartesianChart](Selection_images/maui_multi_seriesselection.PNG)
+![Multiple DataPointSelection support in MAUI SfCartesianChart](Selection_images/maui_multi_dataPoint_selection.png)
+
+N> The Multiple Selection in [DataPointSelectionBehavior]() can also be performed by using the ChartSeries SelectionBehavior property.
+
+## ClearSelection Method
+
+Both the SelectionBehavior has a public method called [ClearSelection ()](), which resets all current Selection Behavior property values to their default values and resets the color of the selected element to default.
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Selection.md:174]: Resets all the selection customization to default. The following code example shows how to use [ClearSelection]().

File: MAUI/Cartesian-Charts/Selection.md [hunk 4]
```diff
@@ -84,44 +87,126 @@ Cartesian chart provides support to select a point programmatically on a chart u
...
-this.Content = chart;
+
+selection.ClearSelection();
 
 {% endhighlight %}
 
 {% endtabs %}
 
-![Initial selection rendering support in MAUI Chart](Selection_images/maui_chart_selection_with_index.png)
+N> The [ClearSelection ()]() method can also be performed in [SeriesSelectionBehavior]() by using the SfCartesianChart SelectionBehavior property.
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Selection.md:194]: remove the code.

**Review by Saravanan-Madhesh — CHANGES_REQUESTED** [review]

File: MAUI/Cartesian-Charts/Selection.md (hunk 4 above)
Comment (MoneeshramDhanabal) [reply MAUI/Cartesian-Charts/Selection.md]: removed the code.

File: MAUI/Cartesian-Charts/Selection.md (hunk 3 above)
Comment (MoneeshramDhanabal) [reply MAUI/Cartesian-Charts/Selection.md]: Changed the content as mentioned.

File: MAUI/Cartesian-Charts/Selection.md [hunk 5]
```diff
@@ -84,44 +87,126 @@ Cartesian chart provides support to select a point programmatically on a chart u
...
+
+{% endtabs %}
+
+![Series Selection support in MAUI SfCartesianChart](Selection_images/maui_cartesianchart_series_selection.PNG)
+
+## Properties
+
+The following properties are common for both Selection Behaviors in [SfCartesianChart]() and [ChartSeries](), and the Selection feature can be configured using the following properties:
+
+* [Type]() - Gets or sets the ChartSelectionType Enum value for the Selection Behavior.
```
[the remark this answers is not in the document]
Comment (MoneeshramDhanabal) [reply MAUI/Cartesian-Charts/Selection.md]: Changed the content as mentioned.

File: MAUI/Cartesian-Charts/Selection.md [hunk 6]
```diff
@@ -84,44 +87,126 @@ Cartesian chart provides support to select a point programmatically on a chart u
...
+{% endhighlight %}
+
+{% endtabs %}
+
+![Multiple SeriesSelection support in MAUI SfCartesianChart](Selection_images/maui_multi_seriesselection.PNG)
+![Multiple DataPointSelection support in MAUI SfCartesianChart](Selection_images/maui_multi_dataPoint_selection.png)
+
+N> The Multiple Selection in [DataPointSelectionBehavior]() can also be performed by using the ChartSeries SelectionBehavior property.
+
+## ClearSelection Method
```
[the remark this answers is not in the document]
Comment (MoneeshramDhanabal) [reply MAUI/Cartesian-Charts/Selection.md]: Changed the content as mentioned.

File: MAUI/Cartesian-Charts/Selection.md [hunk 7]
```diff
@@ -84,44 +87,126 @@ Cartesian chart provides support to select a point programmatically on a chart u
...
+chart.Series.Add(series2);
+chart.Series.Add(series3);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Series Selection support in MAUI SfCartesianChart](Selection_images/maui_cartesianchart_series_selection.PNG)
+
+## Properties
```
[the remark this answers is not in the document]
Comment (MoneeshramDhanabal) [reply MAUI/Cartesian-Charts/Selection.md]: Changed the content as mentioned.

File: MAUI/Cartesian-Charts/Selection.md [hunk 8]
```diff
@@ -68,14 +66,19 @@ Cartesian chart provides support to select a point programmatically on a chart u
 <chart:SfCartesianChart>
 . . .
     <chart:SfCartesianChart.SelectionBehavior>
-        <chart:ChartSelectionBehavior/>
+        <chart:SeriesSelectionBehavior/>
```
[the remark this answers is not in the document]
Comment (MoneeshramDhanabal) [reply MAUI/Cartesian-Charts/Selection.md]: Changed the content as mentioned.

File: MAUI/Cartesian-Charts/Selection.md [hunk 9]
```diff
@@ -37,29 +38,26 @@ To enable the selection in chart, create an instance of [ChartSelectionBehavior]
...
 {% endtabs %}
 
-![Segment selection support in MAUI Chart](Selection_images/maui_chart_segment_selection.png)
+![Segment selection support in MAUI SfCartesianChart](Selection_images/maui_cartesianchart_segment_selection.PNG)
 
-## Selection on initial rendering
+## Enable SeriesSelection
 
-Cartesian chart provides support to select a point programmatically on a chart using the [SelectedIndex](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartSeries.html#Syncfusion_Maui_Charts_ChartSeries_SelectedIndex) property of series.
+To enable the series selection in the [SfCartesianChart](), create an instance of [SeriesSelectionBehavior]() and specify a color value for the [SelectionBrush]() property to highlight the entire selected series. The selection instance should then be set to the SfCartesianChart [SelectionBehavior]() property.
```
[the remark this answers is not in the document]
Comment (MoneeshramDhanabal) [reply MAUI/Cartesian-Charts/Selection.md]: Changed the content as mentioned.

File: MAUI/Cartesian-Charts/Selection.md [hunk 10]
```diff
@@ -37,29 +38,26 @@ To enable the selection in chart, create an instance of [ChartSelectionBehavior]
...
 
 {% endhighlight %}
 
 {% endtabs %}
 
-![Segment selection support in MAUI Chart](Selection_images/maui_chart_segment_selection.png)
+![Segment selection support in MAUI SfCartesianChart](Selection_images/maui_cartesianchart_segment_selection.PNG)
 
-## Selection on initial rendering
+## Enable SeriesSelection
```
[the remark this answers is not in the document]
Comment (MoneeshramDhanabal) [reply MAUI/Cartesian-Charts/Selection.md]: Changed the spacing as mentioned.

File: MAUI/Cartesian-Charts/Selection.md [hunk 11]
```diff
@@ -7,28 +7,29 @@ control: SfCartesianChart
...
 ---
 
-# Selection in .NET MAUI Chart
+# Selection in .NET MAUI SfCartesianChart
 
-Cartesian chart supports selection that allows to select or highlight a segment in the chart by using [ChartSelectionBehavior](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartSelectionBehavior.html).
+[SfCartesianChart]() provides selection behavior support, which allows you to select or highlight a segment in a series or a series in the Chart using the [DataPointSelectionBehavior]() or [SeriesSelectionBehavior]().
 
-## Enable Selection
+## Enable DataPointSelection
```
[the remark this answers is not in the document]
Comment (MoneeshramDhanabal) [reply MAUI/Cartesian-Charts/Selection.md]: Changed the spacing as mentioned.

File: MAUI/Cartesian-Charts/Selection.md [hunk 12]
```diff
@@ -7,28 +7,29 @@ control: SfCartesianChart
 documentation: ug
 ---
 
-# Selection in .NET MAUI Chart
+# Selection in .NET MAUI SfCartesianChart
```
[the remark this answers is not in the document]
Comment (MoneeshramDhanabal) [reply MAUI/Cartesian-Charts/Selection.md]: Changed the title.

File: MAUI/Cartesian-Charts/Selection.md [hunk 13]
```diff
@@ -7,28 +7,29 @@ control: SfCartesianChart
...
-    <chart:ColumnSeries ItemsSource="{Binding Data}" 
-                        XBindingPath="Demand"
-                        YBindingPath="Year2010"
-                        SelectionBrush="Green"/>
+    <chart:SfCartesianChart.Series>
+        <chart:ColumnSeries ItemsSource="{Binding Data}" 
+                        XBindingPath="Time"
+                        YBindingPath="FootStepsCount">
+                <chart:ColumnSeries.SelectionBehavior>
+                        <chart:DataPointSelectionBehavior/>
```
[the remark this answers is not in the document]
Comment (MoneeshramDhanabal) [reply MAUI/Cartesian-Charts/Selection.md]: Added Selection Brush

File: MAUI/Cartesian-Charts/Selection.md [hunk 14]
```diff
@@ -7,28 +7,29 @@ control: SfCartesianChart
 documentation: ug
 ---
 
-# Selection in .NET MAUI Chart
+# Selection in .NET MAUI SfCartesianChart
 
-Cartesian chart supports selection that allows to select or highlight a segment in the chart by using [ChartSelectionBehavior](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartSelectionBehavior.html).
+[SfCartesianChart]() provides selection behavior support, which allows you to select or highlight a segment in a series or a series in the Chart using the [DataPointSelectionBehavior]() or [SeriesSelectionBehavior]().
```
[the remark this answers is not in the document]
Comment (MoneeshramDhanabal) [reply MAUI/Cartesian-Charts/Selection.md]: Changed as mentioned.

File: MAUI/Cartesian-Charts/Selection.md [hunk 15]
```diff
@@ -7,28 +7,29 @@ control: SfCartesianChart
...
 
-## Enable Selection
+## Enable DataPointSelection
 
-To enable the selection in chart, create an instance of [ChartSelectionBehavior](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartSelectionBehavior.html) and set it to the [SelectionBehavior](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartBase.html#Syncfusion_Maui_Charts_ChartBase_SelectionBehavior) property of [SfCartesianChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1). And also need to set the `SelectionBrush` property to highlight the segment in the chart.
+To enable the data point selection in the [SfCartesianChart](), create an instance of [DataPointSelectionBehavior]() and specify a color value for the [SelectionBrush]() property to highlight the selected segment in the series. The selection instance should then be set to the chart series [SelectionBehavior]() property.
```
[the remark this answers is not in the document]
Comment (MoneeshramDhanabal) [reply MAUI/Cartesian-Charts/Selection.md]: Changed as mentioned.

File: MAUI/Cartesian-Charts/Selection.md [hunk 16]
```diff
@@ -37,29 +38,27 @@ To enable the selection in chart, create an instance of [ChartSelectionBehavior]
...
 
 {% endhighlight %}
 
 {% endtabs %}
 
-![Segment selection support in MAUI Chart](Selection_images/maui_chart_segment_selection.png)
+![Segment selection support in MAUI SfCartesianChart](Selection_images/maui_cartesianchart_segment_selection.PNG)
 
-## Selection on initial rendering
+## Enable SeriesSelection
```
[the remark this answers is not in the document]
Comment (MoneeshramDhanabal) [reply MAUI/Cartesian-Charts/Selection.md]: Given the spacing as mentioned

**Comment by Saravanan-Madhesh** [comment]
Changes were merged. 
https://github.com/syncfusion-content/maui-docs/pull/607

**Pull request closed without merging**